### ISSUE TITLE
install package sjemea

### ISSUE THREAD

**Issue by TsungChihTsai** [CCutts/Detecting_pairwise_correlations_in_spike_trains#2]
Because [processing_functions.R](https://github.com/CCutts/Detecting_pairwise_correlations_in_spike_trains/blob/master/processing_functions.R) require sjemea

And, I attempted to use " install.packages("devtools")
devtools::install_github("sje30/sjemea")" 
This installation command is from "https://github.com/sje30/sjemea"

But I still have problem that 

"> devtools::install_github("sje30/sjemea")
Downloading GitHub repo sje30/sjemea@HEAD
trying URL 'https://cran.rstudio.com/bin/windows/Rtools/rtools43/files/rtools43-5863-5818.exe'
Content type 'application/x-msdownload' length 451265016 bytes (430.4 MB)
downloaded 430.4 MB

Error: Failed to install 'sjemea' from GitHub:
  Could not find tools necessary to compile a package
Call `pkgbuild::check_build_tools(debug = TRUE)` to diagnose the problem.

**Comment by bxy666666** [comment]
Hello, may I ask if you have resolved it?

**Comment by TsungChihTsai** [comment]
> Hello, may I ask if you have resolved it?

No, I tried a lot of ways to resolve and saw other GitHub issue. But still not working. Do you have any idea?
